diff --git a/www/js/index.js b/www/js/index.js
index 6fc66e5..cf4f4cd 100644
--- a/www/js/index.js
+++ b/www/js/index.js
@@ -23,7 +23,7 @@ document.addEventListener('deviceready', onDeviceReady, false);
 
 function onDeviceReady() {
     // Cordova is now initialized. Have fun!
-
+    TestFairy.begin("APP TOKEN");
     console.log('Running cordova-' + cordova.platformId + '@' + cordova.version);
     document.getElementById('deviceready').classList.add('ready');
 }
