diff --git a/src/main.ts b/src/main.ts
index 91ec6da..93a7fc8 100644
--- a/src/main.ts
+++ b/src/main.ts
@@ -4,6 +4,9 @@ import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { AppModule } from './app/app.module';
 import { environment } from './environments/environment';
 
+declare var TestFairy: any;
+TestFairy.begin("5b3af35e59a1e074e2d50675b1b629306cf0cfbd");
+
 if (environment.production) {
   enableProdMode();
 }
